pulsar-broker/src/test/java/org/apache/pulsar/client/api/DeadLetterTopicTest.java

```diff
@@ -794,4 +795,16 @@ public void testDeadLetterTopicWithInitialSubscriptionAndMultiConsumers() throws
         otherConsumer.close();
         consumer.close();
     }
+
+    @Test
+    public void testDeadLetterPolicyDeserialize() throws Exception {
+        ConsumerBuilder<String> consumerBuilder = pulsarClient.newConsumer(Schema.STRING);
+        DeadLetterPolicy policy =
+                DeadLetterPolicy.builder().deadLetterTopic("a").retryLetterTopic("a").initialSubscriptionName("a")
+                        .maxRedeliverCount(1).build();
+        consumerBuilder.deadLetterPolicy(policy);
```

Comment:
Do you want to add policy in the config map?

Reply:
I want test DeadLetterPolicy deserialization

Reply:
Maybe you need to add the `DeadLetterPolicy` in the config map?

Reply:
Got it.